### PR TITLE
Change behaviour for activation/deactivation of a field in a document type

### PR DESCRIPTION
**Pull request by poi33**
Had to change the stylesheet building to use css 
Added striketrough styling to documentfType field
Added active toggle button to the edit document field modal

Note: It is currently only possible to toggle the state of a field in code. This functionality needs to be added to the fields?

### CODE REVIEW

File: src/main/resources/assets/style/main.sass [hunk 1]
```diff
@@ -1,88 +1,11 @@
+@use 'components'
+
+// Global variables used by css (not compile time)
+:root
+	--normal-text-size: 14px
```
Comment (ComLock) [inline src/main/resources/assets/style/main.sass:5]: This seems to be a global css change... is that safe???

**Comment by ComLock** [comment]
The active toggle doesn't work?
The active toggle is visible in the dialog for global fields too...

**Comment by ComLock** [comment]
Yeah, let's look this over IRL

**Review by ComLock — CHANGES_REQUESTED** [review]

File: src/main/resources/assets/style/main.sass (hunk 1 above)
Comment (poi33) [reply src/main/resources/assets/style/main.sass:5]: Yes css variables are safe

**Comment by poi33** [comment]
@ComLock should be updated with proper active deactivate state now

File: src/main/resources/assets/react/documentTypes/AddOrEditLocalFieldModal.jsx [hunk 2]
```diff
@@ -193,9 +193,7 @@ export const AddOrEditLocalFieldModal = ({
 							// labelPosition= {'right'}
 							toggle
 							onChange={()=>{
-								/* Currently ignored */
-								//setActive(!active)
-								console.log('Create active state');
+								setActive(!active);
```
Comment (ComLock) [inline src/main/resources/assets/react/documentTypes/AddOrEditLocalFieldModal.jsx:196]: I guess this should work, in that you negate the previous state. I usually get the new value from the onChange event.

**Review by ComLock — APPROVED** [review]

**Pull request merged**